graphql_compiler/query_formatting/graphql_formatting.py

```diff
@@ -21,43 +57,7 @@ def fix_indentation_depth(query):
             raise AssertionError(u'Indentation was not a multiple of two: '
                                  u'{}'.format(consecutive_spaces))
 
-        final_lines.append(('  ' * consecutive_spaces) + line[consecutive_spaces:])
+        final_lines.append(('  ' * consecutive_spaces) +
+                           line[consecutive_spaces:])
 
     return '\n'.join(final_lines)
-
-
-def fix_filter_directive_order(query):
-    """Reverse the order of filter arguments. GraphQL insists on printing them backwards."""
-    filter_directive_pattern = (
-        r'@filter\('
-        r'value: \['
-        r'("[$%][a-zA-Z0-9_]+"'         # start of capture group 1 at start of this line
-        r'(?:, "[$%][a-zA-Z0-9_]+")*)'  # end of capture group 1 at end of this line
-        r'\], '
-        r'op_name: '
-        r'("[^"]+")'                    # capture group 2 on this line
-        r'\)'
-    )
-
-    replacement_pattern = (
-        r'@filter(op_name: \2, value: [\1])'
-    )
-
-    return re.sub(filter_directive_pattern, replacement_pattern, query)
-
-
-def pretty_print_graphql(query, use_four_spaces=True):
```

Comment:
Why not use and amend this existing function, since it already has detailed comments?